reri_err_reporting.adoc

```diff
@@ -178,21 +177,31 @@ The `bank_info` is a read-only register and its layout is as follows:
 ....
 {reg: [
   {bits: 16, name: 'inst_id'},
-  {bits: 16, name: 'n_err_recs'},
-  {bits: 24, name: 'WPRI'},
+  {bits:  6, name: 'n_err_recs'},
+  {bits:  2, name: 'layout'},
+  {bits: 32, name: 'WPRI'},
   {bits:  8, name: 'version'},
 ], config:{lanes: 2, hspace:1024}}
 ....
 
 The `version` field returns the version of the architectural register layout
 specification implemented by the error bank. The version defined by this
-specification is 0x01.
+specification is 0x01. The encodings 0xF? of this field are designated for
+custom use.
```

Comment:
The encoding of 0xF of this field is designated for custom use.

Reply:
Its 0xF? - where the lower nibble is a wildcard.

Reply:
The updated text states it more explicitly.